x-pack/plugins/security_solution/public/timelines/components/edit_data_provider/helpers.test.tsx

```diff
@@ -204,6 +219,25 @@ describe('helpers', () => {
               label: 'invalid-operator',
             },
           ],
+          type: DataProviderType.default,
+        })
+      ).toBe(false);
+    });
+    test('it should return false when the selected operator is "is one of", and the DataProviderType is template', () => {
```

Comment:
consider adding an additional test for the `is not one of` operator

Reply:
added!